engine/graphics/src/graphics.h

```diff
@@ -632,7 +634,26 @@ namespace dmGraphics
     uint16_t    GetMipmapSize(uint16_t size_0, uint8_t mipmap);
     uint8_t     GetMipmapCount(uint16_t size);
 
-    bool IsAssetHandleValid(HContext context, HAssetHandle asset_handle);
+
+    // Asset handle helpers
+    const char* GetAssetTypeLiteral(AssetType type);
+    bool        IsAssetHandleValid(HContext context, HAssetHandle asset_handle);
+
+    static inline HAssetHandle MakeAssetHandle(HOpaqueHandle opaque_handle, AssetType asset_type)
+    {
+        assert(asset_type != ASSET_TYPE_NONE && "Invalid asset type");
+        return ((uint64_t) asset_type) << 32 | opaque_handle;
```

Comment:
I kind of expected a chane in the other PR, where we _do_ check that we don't use more than 53 bits?
We want to catch is as early as possible, right?

ALos in `IsAssetHandleValid()`